src/ray/gcs/gcs_server/gcs_resource_scheduler.h

```diff
@@ -48,8 +49,9 @@ enum class SchedulingResultStatus {
   SUCCESS = 2,
 };
 
-typedef std::pair<NodeID, double> NodeScore;
-typedef std::pair<SchedulingResultStatus, std::vector<NodeID>> SchedulingResult;
+typedef std::pair<scheduling::NodeID, double> NodeScore;
```

Comment:
use `using NodeScore = `

Reply:
> use `using NodeScore = `

Okay, if possible, I'll fix this in the next PR。

Reply:
Why next PR and not this PR?
Btw, I think we should consider adding it to cpp standards, cc @scv119 
Some reference [here](https://www.nextptr.com/tutorial/ta1193988140/how-cplusplus-using-or-aliasdeclaration-is-better-than-typedef)